src/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Leaderboard</title>
+</head>
+<body>
+  <header class="header">
+    <h1>Leaderboard</h1>
+  </header>
+  <main class="main">
+    <section class="scores">
+      <div class="title">
+        <h2>Recent scores</h2>
+        <button id="refreshBtn" type="button">Refresh</button>
+      </div>
+
+      <ul class="score-list">
+        <li>Name: 100</li>
+        <li>Name: 20</li>
+        <li>Name: 50</li>
+      </ul>
+    </section>
+
+    <section class="add-score">
+      <h2>Add your score</h2>
+
+      <form action="#">
+        <input type="text" id="name" placeholder="Your name" autocomplete="off">
+        <input type="text" id="score" placeholder="Your score" autocomplete="off">
+        <button type="submit" id="submitBtn">Submit</button>
+      </form>
+    </section>
+  </main>
```

Comment:
- [ ] It looks like you add everything directly on the `index.html` file. In this case, try to use index.html main contain an only root element (in this case you use `<script src="main.js"></script>`). After you called root, try to add all this on the HTML file using js. Remember to follow all ES6 best practices.